stories/dashboard-details.stories.js

```diff
@@ -56,7 +52,12 @@ storiesOf( 'Dashboard Details', module )
 
 		return (
 			<WithTestRegistry callback={ setupRegistry }>
-				<DashboardDetailsApp />
+				<FeaturesProvider value={ {
+					widgets: { pageDashboard: true },
+					storeErrorNotifications: true,
```

Comment:
Per previous comments, would this be more intuitive as `[ 'widgets.pageDashboard', 'storeErrorNotifications' ]` instead?